### DISCUSSION TITLE
How To Place Text Blocks in Image

### DISCUSSION THREAD

**Discussion by vindhya124** [PiranhaCMS/piranha.core#1938] [General] [answered]
Is there a way we can place a text on image?. Please find below image for reference.

![image](https://user-images.githubusercontent.com/38023844/192899693-9b9bae21-f0c2-4c79-91aa-edf4e0d42689.png)

**Comment by tidyui** [comment, marked as answer]
Hi there! You create a custom block that has an ImageField and a HtmlField. https://piranhacms.org/docs/master/content/blocks/custom-blocks

Regards

**Reply by vindhya124** [reply]
Got it. Thank you
